src/core/Akka.Remote.TestKit/MsgEncoder.cs

```diff
@@ -74,12 +74,15 @@ public void Encode(IConnection connection, object message, out List<IByteBuf> en
                                 .SetOp(BarrierOp.Fail)))
                     .With<ThrottleMsg>(
                         throttle =>
+                        {
                             w.SetFailure(
                                 InjectFailure.CreateBuilder()
                                     .SetFailure(TCP.FailType.Throttle)
                                     .SetAddress(Address2Proto(throttle.Target))
+                                    .SetFailure(TCP.FailType.Throttle)
```

Comment:
Fixes some bugs with using throttling inside the MNTR - we weren't encoding all of the fields into the protobuf message correctly before.